ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/LmdGhost.java

```diff
@@ -38,20 +44,20 @@
    * @param start_state
    * @param start_block
    * @return
-   * @throws StateTransitionException
    */
   public static BeaconBlock lmd_ghost(
-      ChainStorageClient store, BeaconState start_state, BeaconBlock start_block)
-      throws StateTransitionException {
+      ChainStorageClient store, BeaconState start_state, BeaconBlock start_block) {
+    List<Validator> validators = start_state.getValidator_registry();
     List<Integer> active_validator_indices =
         ValidatorsUtil.get_active_validator_indices(
-            start_state.getValidator_registry(),
-            BeaconStateUtil.slot_to_epoch(UnsignedLong.valueOf(start_block.getSlot())));
+            validators, slot_to_epoch(UnsignedLong.valueOf(start_block.getSlot())));
```

Comment:
In the spec it is start_state.getSlot(), not sure if it makes a difference but if you want to align with the spec it would be better to change.

Reply:
good catch!